tox.ini

```diff
@@ -89,3 +90,8 @@ commands =
     !notebooks: sphinx-build -j 1 --color -W --keep-going -b html -d _build/.doctrees . _build/html -D nb_execution_mode=off {posargs}
     notebooks: sphinx-build -j 1 --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
     python -c 'import pathlib; print("Documentation available under file://\{0\}".format(pathlib.Path(r"{toxinidir}") / "docs" / "_build" / "index.html"))'
+
+[testenv:benchmarks]
+description = Run benchmarks against latest release to ensure there are no performance regressions
+commands =
+    {env:PYTEST_COMMAND} -k test_benchmarks --benchmark-autosave --benchmark-compare=0001
```

Comment:
I propose that instead of `-k test_benchmarks` we put the [`@pytest.mark.benchmark`](https://pytest-benchmark.readthedocs.io/en/latest/usage.html#markers) marker on all our benchmarks and then do `-m benchmark` to select tests with that marker.